Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -668,9 +668,15 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 			}
 		}
 
-        public static GameObject GetGameObjectByNetworkId(uint id)
+		/// <summary>
+		/// A helper function to retrieve a GameObject by its network id.
+		/// </summary>
+		/// <param name="id">Network id of the gameobject</param>
+        public GameObject GetGameObjectByNetworkId(uint id)
         {
-            return ((NetworkBehavior)Instance.Networker.NetworkObjects[id].AttachedBehavior).gameObject;
+			NetworkObject foundNetworkObject;
+			Networker?.NetworkObjects?.TryGetValue(id, out foundNetworkObject);
```

Comment:
This is unfortunatelly a C# 6 feature that would break for users not having that available for them.

Reply:
Ah lol. I didn't think my commits would automatically become part of the PR. I didn't have my main machine, so was going to squash the commit after testing. 

I'll take them out, though.

Reply:
yeah if you push changes to a branch that has a PR then the PR will automatically have the changes 😉